### ISSUE TITLE
TypeLoadException in patcher

### ISSUE THREAD

**Issue by drok** [sway2020/YetAnotherToolbar#22]
[Harmony 0.9] 2448994345 threw: TypeLoadException: Could not load type `YetAnotherToolbar.Patcher` from assembly `YetAnotherToolbar, Version=1.1.2.0, Culture=neutral, PublicKeyToken=null`.

```
13 * 2448994345               2448994345                       2448994345                       CH: 2.0.4.0    2.0.0.0   
                   [ERR] Exceptions thrown: 19
                   [ERR]   TypeLoadException: Could not load type 'YetAnotherToolbar.Patcher' from assembly 'YetAnotherToolbar, Version=1.1.2.0, Culture=neutral, PublicKeyToken=null'. (see https://github.com/drok/Harmony-CitiesSkylines/issues/9): 19 times
```

I think you haven't tested the case where a first time user of your mod loads it up without having any Harmony present. Your mod would just crash instead of offering to install the library.

The issue reported at drok/Harmony-CitiesSkylines/issues/9 not only explains the problem, but also contains links to other commits where mod authors have fixed this exact bug in their mods.

I have noted that you expect a pull request. I think I've done more than enough to pinpoint the cause of the bug.

End user's log file: https://pastebin.pl/view/d0c52a38

**Comment by iansui** [comment]
I'll take a look at this on the weekend and see if I can solve it

**Comment by sway2020** [comment]
I'll take a look at this on the weekend and try to solve it

**Comment by sway2020** [comment]
Fixed in v1.1.3. The update was just pushed onto the workshop
